forte/data/data_pack.py

```diff
@@ -767,11 +767,13 @@ def validate_group(cls, entry: EntryType) -> bool:
 
     def get_data(
         self,
-        context_type: Union[str, Type[Annotation]],
+        context_type: Union[str, Type[Annotation], Type[AudioAnnotation]],
         request: Optional[DataRequest] = None,
         skip_k: int = 0,
     ) -> Iterator[Dict[str, Any]]:
-        r"""Fetch entries from the data_pack of type `context_type`.
+        r"""Fetch primitive data from entries in the data_pack of type
+        `context_type`. Primitive data includes `"span"`, annotation-specifc
```

Comment:
The sentence is hard to read, what are `annotation specific default data fields`? And there is a misspelling here.

We don't have to emphasize "primitive data", in fact it is not accurate to say this way.

And then, what do you mean by `fetch data of type context_type`? We are also getting data from `requests`. So the docstring is not really accurate right now

Reply:
I saw you try to explain them later, maybe you should write "see below" to guide readers.